code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -267,30 +267,34 @@
 
 /**
  * InsertID
- * Attempt to insert the ID in either card slot.
+ * Attempt to insert the ID in either card slot, if ID is present - attempts swap
  * Args:
  * inserting_id - the ID being inserted
  * user - The person inserting the ID
  */
 /obj/item/modular_computer/InsertID(obj/item/card/inserting_id, mob/user)
-	//all slots taken
-	if(computer_id_slot)
+	if(!isnull(user) && !user.transferItemToLoc(inserting_id, src))
 		return FALSE
 
-	if(user)
-		if(!user.transferItemToLoc(inserting_id, src))
-			return FALSE
-		to_chat(user, span_notice("You insert \the [inserting_id] into the card slot."))
 	else
 		inserting_id.forceMove(src)
 
+	if(!isnull(computer_id_slot))
+		RemoveID(user, TRUE)
```

Comment:
```suggestion
		RemoveID(user, silent = TRUE)
```

it's nice that you added the argument for stuff like this but it's not intuitive for people to know what `TRUE` would mean in this context, so let's name this argument.

Reply:
Sure, it's a neat remark.